cmd/buildah/common.go

```diff
@@ -57,3 +59,27 @@ func openBuilder(store storage.Store, name string) (builder *buildah.Builder, er
 func openBuilders(store storage.Store) (builders []*buildah.Builder, err error) {
 	return buildah.OpenAllBuilders(store)
 }
+
+/****
+* Returns up to the first two names in the slice whose value starts with
+* the same value as the shortName.
+*****/
+func findShortName(names []string, shortName string) []string {
```

Comment:
Yes keep it out, and add it to the kpod rm and kpod rmi, patch set you are working on.